dart/test/hub_test.dart

```diff
@@ -95,6 +95,14 @@ void main() {
       final returnedId = await hub.captureEvent(event);
       expect(eventId.toString(), returnedId.toString());
     });
+
+    test('should install integrations', () {
+      options.addIntegration(integration);
+      final hub = Hub(options);
+      client = MockSentryClient();
+      hub.bindClient(client);
+      // TODO: assert that integration(hub, options) has been invoked
```

Comment:
@rxlabz do you know how to mock a type function?
I need to verify that the integration list has been executed